site/components/layout/HomeLayout.js

```diff
@@ -0,0 +1,22 @@
+import { inter } from '../../fonts'
+import Navbar from '../Navbar'
+
+import Footer from './Footer'
+import Head from './Head'
+import Hero from './Hero'
+
+const HomeLayout = ({ children, title }) => (
+  <>
+    <Head title={title} />
+    <div className={`w-full ${inter.className}`}>
+      <div className="">
+        <Navbar />
+        <Hero />
+        <div className="container">{children}</div>
+        <Footer />
+      </div>
```

Comment:
empty `className` can be removed. 

```suggestion
      <div>
        <Navbar />
        <Hero />
        <div className="container">{children}</div>
        <Footer />
      </div>
```

I also wonder if then, the whole div can be removed

```suggestion
        <Navbar />
        <Hero />
        <div className="container">{children}</div>
        <Footer />
```

Reply:
done